src/Services/FHIR/FhirResourcesService.php

```diff
@@ -29,7 +29,7 @@ public function createBundle($resource = '', $resource_array = [], $encode = tru
         // set bundle type default to collection so may include different
         // resource types. at least I hope thats how it works....
         $bundleInit = array(
-            'identifier' => $resource . "bundle",
+            // 'identifier' => $resource . "bundle",
```

Comment:
this looks very reasonable since agree identifier not needed in search:
https://www.hl7.org/fhir/bundle-definitions.html#Bundle.identifier

(rec commenting this out along with a quick description and above link for why commented out in case in future we support bundles that would need a identifier)

Reply:
Added the comment and the link.